src/Imagine/Cache/CacheManager.php

```diff
@@ -145,9 +145,9 @@ public function isStored(string $path, string $filter, ?string $resolver = null)
      *
      * @throws NotFoundHttpException if the path can not be resolved
      *
-     * @return string The url of resolved image
+     * @return string|null The url of resolved image
```

Comment:
is the reason that this is nullable the signature of $postEvent->getUrl()? maybe it would be better to make that one have to return a string.

it seems better to me to either return an url or throw an exception, rather than the caller having to verify if it got back an url.